test/NuGet.Core.Tests/NuGet.Protocol.Tests/HttpSource/HttpSourceAuthenticationHandlerTests.cs

```diff
@@ -534,6 +534,22 @@ public async Task SendAsync_RetryWithClonedPostRequest(HttpContent httpContent)
             Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
         }
 
+        [Fact]
+        public void Dispose_CallingDisposeWithoutArguments_DisposesInstance()
```

Comment:
"WithoutArguments" really means `isDisposing:true` when Dispose gets called.

Reply:
True, but I want to test methods that are public.

Reply:
Removed test.